pjlib/src/pj/log.c

```diff
@@ -299,8 +300,9 @@ static void resume_logging(int *saved_level)
 #if PJ_HAS_THREADS
     if (thread_suspended_tls_id != -1) 
     {
-        pj_thread_local_set(thread_suspended_tls_id,
-                            (void*)(pj_size_t)PJ_FALSE);
+        static pj_ssize_t suspend_log = PJ_FALSE;
+        pj_thread_local_set(thread_suspended_tls_id, 
+                            (void*)&suspend_log);
```

Comment:
As now the TLS values have been changed, `is_logging_suspended()` needs to be updated too, something like:
```
return *((pj_ssize_t*)pj_thread_local_get(thread_suspended_tls_id)) == PJ_TRUE;
```

Reply:
Good catch!
It can be NULL though, the first time `pj_log()` is called.